test/countries/test_spain.py

```diff
@@ -265,6 +206,115 @@ def test_variable_days_in_2022(self):
                     % (date(2022, *fest_date), prov, ", ".join(fest_provs)),
                 )
 
+    def test_variable_days_in_2023(self):
+        province_days = {
+            (1, 2): [
+                "AN",
+                "AR",
+                "AS",
+                "CL",
+                "MC",
+            ],
+            (1, 6): [
```

Comment:
I believe it's okay to use a set here too (and below)

Reply:
Done